docs/man/rpmdeps.1.scd

```diff
@@ -1,50 +1,76 @@
 RPMDEPS(1)
 
 # NAME
-
 rpmdeps - Generate RPM Package Dependencies
 
 # SYNOPSIS
+*rpmdeps* [options]
 
-*rpmdeps* [options] {*-P*|*--provides*} {*-R*|*--requires*} [_FILE_ ...]
+*rpmdeps* [options] _FILE_ ...
 
 # DESCRIPTION
+*rpmdeps* generates package dependencies for one or more files, received
+either as a list of filenames on the standard input, or command line
+arguments.
+
+*rpmdeps* is mainly useful for troubleshooting dependency generation
+and developing new dependency generators.
+
+Note that it is not actually used by *rpmbuild*(1) for automatic
+dependency generation except in a legacy compatibility mode.
 
-*rpmdeps* generates package dependencies for the set of _FILE_
-arguments. Each _FILE_ argument is searched for Elf32/Elf64, script
-interpreter, or per-script dependencies, and the dependencies are
-printed to stdout.
+# ARGUMENTS
+_FILE_
+	Any regular file.
```

Comment:
I think we should also add a note about the no-argument variant (much like e.g. in the updated rpmarchive man page).